package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hyperproof/hypersync-sdk",
-  "version": "0.8.1",
+  "version": "0.8.3",
```

Comment:
When I published 0.8.2 to npm I failed to update GH.  Still working on my workflow.